strapi/strapi
# [fix] i18n entity-service decorator findMany

### What does it do?

It fixes the findMany decorator to use other decorators
### Why is it needed?

So that other entity service decorators can be used with i18n

### How to test it?

Run it with a findMany decorator bellow it and see that the output does now match what you expect aka the lower decorators are called

### Related issue(s)/PR(s)
